src/js/_enqueues/wp/password-strength-meter.js

```diff
@@ -41,28 +41,28 @@ window.wp = window.wp || {};
 				return -1;
 			}
 
-			var result = zxcvbn( password1, blacklist );
+			var result = zxcvbn( password1, blocklist );
 			return result.score;
 		},
 
 		/**
 		 * Builds an array of words that should be penalized.
 		 *
 		 * Certain words need to be penalized because it would lower the entropy of a
-		 * password if they were used. The blacklist is based on user input fields such
+		 * password if they were used. The blocklist is based on user input fields such
 		 * as username, first name, email etc.
 		 *
 		 * @since 3.7.0
 		 *
-		 * @return {string[]} The array of words to be blacklisted.
+		 * @return {string[]} The array of words to be blocklisted.
 		 */
-		userInputBlacklist : function() {
+		userInputBlocklist : function() {
```

Comment:
Plugins need to call this directly to use the strength meter properly in signup forms, https://wpdirectory.net/search/01EB4KTBNMTDFFW6SQMV4A37CE. We could maintain BC by keeping the old function name, aliasing it to the new one, and using the `wp.deprecated.deprecated`, https://github.com/WordPress/gutenberg/tree/master/packages/deprecated.

Reply:
Thanks! Implemented this.